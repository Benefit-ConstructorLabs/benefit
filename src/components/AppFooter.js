import React from 'react';
import '../../styles/components/app-footer.scss';

const AppFooter = () => (
  <footer>
    <p className="copyright">&copy; 2018 Benefit Team</p>
  </footer>
);

export default AppFooter;
