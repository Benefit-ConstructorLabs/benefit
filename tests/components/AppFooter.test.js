import React from 'react';
import renderer from 'react-test-renderer';
import AppFooter from '../../src/components/AppFooter';


describe('AppFooter', () => {
  test('should render correctly', () => {
    const tree = renderer
      .create(<AppFooter />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
