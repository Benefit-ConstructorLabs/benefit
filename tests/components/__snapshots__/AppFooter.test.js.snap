// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppFooter should render correctly 1`] = `
<footer>
  <p
    className="copyright"
  >
    Â© 2018 Better Change Team
  </p>
</footer>
`;
